README.md

```diff
@@ -31,8 +31,9 @@ for some tooling scripts. The code itself is built using Bazel.
 
 #### Set up `dev-env` on Linux or macOS
 
-1. Install Nix: `bash <(curl https://nixos.org/nix/install)`
-2. Enter `dev-env`: `eval "$(dev-env/bin/dade assist)"`
+1. `cd` into the new `daml` repository you just cloned
```

Comment:
```suggestion
1. Use `cd daml` to switch into the new `daml` repository you just cloned
```
Maybe even clearer on what specific command needs to be executed? Your choice :smile:

Reply:
Removing ambiguity is the name of the game here - good shout